scripts/chain_params.js

```diff
@@ -78,6 +79,7 @@ export const cChainParams = {
         proposalFeeConfirmRequirement: 3,
         maxPaymentCycles: 20,
         maxPayment: 10 * 144 * COIN, // 144 blocks of 10 tPIV
+        defaultColdStakingAddress: 'WmNziUEPyhnUkiVdfsiNX93H6rSJnios44', // Sparrow's Testnet Cold Pool
```

Comment:
Working fine for me (check `Staker` address on my Testnet Stakes).
https://testnet.rockdev.org/xpub/xpub6BnhnQftySgebQbgVQHvgcEZ69nWBXJe75dnw1FkpCasD5CQMiwvMYxRFmhuxJTAyofkRSfqPPHG3u8KrDaMYDWt6Mha1z6SGqHwB7GV6Eq

Also, I don't see a better spot than chainparams, since this is chain-specific and hardcoded.

Reply:
right just wanted to be sure, yeah I also can't see a better spot